### PR TITLE
Fix: Template broken url

### PR DESCRIPTION
**Pull request by chryw**
## Pull request checklist

Make sure you:

- [x] Signed the [Contributor License Agreement](https://cla.js.foundation/webhintio/hint)
- [x] Followed the [commit message guidelines](https://webhint.io/docs/contributor-guide/getting-started/pull-requests/#commit-messages)

For non-trivial changes, please make sure you also:

- [ ] Added/Updated related documentation.
- [ ] Added/Updated related tests.

## Short description of the change(s)

The `create-hint` template has some old/broken urls in the comments. I updated the urls to point to the pages that I think they should:
- about schema: https://webhint.io/docs/contributor-guide/how-to/hint/#the-meta-property
- how to hint: https://webhint.io/docs/contributor-guide/how-to/hint/

Feel free to update the links if I got the pages wrong. Thanks!

### CODE REVIEW

File: packages/create-hint/src/templates/meta.ts.hbs [hunk 1]
```diff
@@ -34,8 +34,8 @@ const meta: HintMetadata = {
         /*
          * If you want to allow the user to configure your hint
          * you should use a valid JSON schema. More info in:
-         * https://webhint.io/docs/contributor-guide/hints/#themetaproperty
-         */
+         * https://webhint.io/docs/contributor-guide/how-to/hint/#the-meta-property
+         *
```
Comment (sarvaje) [inline packages/create-hint/src/templates/meta.ts.hbs]: you need to close the comment :)

**Comment by antross** [comment]
Thanks @chryw! 

Makes me think we need something like our [markdown-link-validator](https://github.com/webhintio/markdown-link-validator) for code comments....

File: packages/create-hint/src/templates/meta.ts.hbs (hunk 1 above)
Comment (chryw) [reply packages/create-hint/src/templates/meta.ts.hbs]: ah, thanks @sarvaje! accidentally deleted a line 😅

**Comment by antross** [comment]
Congrats on your first webhint PR @chryw! 🎉

**Comment by chryw** [comment]
> Thanks @chryw!
> 
> Makes me think we need something like our [markdown-link-validator](https://github.com/webhintio/markdown-link-validator) for code comments....

We actually use it for our team's doc. It's pretty helpful in general. However we just noticed recently that it chokes on some smaller headers like ## or ### below. I'm curious what the problem is.

**Pull request merged**